CMakeLists.txt

```diff
@@ -9,9 +9,9 @@ project(
 # options
 #
 #
-# <INSERT_CMAKE_INSTALL_PREFIX_HERE>
 #set(CMAKE_INSTALL_PREFIX "/opt/poky/rzn1/sysroots/x86_64-pokysdk-linux/usr")
-
+#set(CMAKE_INSTALL_PREFIX "/opt/poky/rzn1/sysroots/x86_64-pokysdk-linux/usr")
```

Comment:
any reason for re-arranging the comment?

Reply:
There was, I can remember why though. I'll move it back to it's OG line in next push